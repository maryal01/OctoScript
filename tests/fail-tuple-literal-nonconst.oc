bool b = false;
(1, b);