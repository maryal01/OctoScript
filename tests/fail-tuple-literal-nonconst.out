Fatal error: exception Failure("Non-constant expression or nested complex type literal found in a tuple literal")